test_runner/batch_others/test_wal_acceptor_async.py

```diff
@@ -0,0 +1,166 @@
+import asyncio
+import aiopg
+import random
+
+from fixtures.zenith_fixtures import WalAcceptor, WalAcceptorFactory, ZenithPageserver, PostgresFactory, Postgres
+from typing import List
+
+pytest_plugins = ("fixtures.zenith_fixtures")
+
+
+class BankClient(object):
+    def __init__(self, conn: aiopg.Connection, n_accounts=None, init_amount=None):
+        self.conn: aiopg.Connection = conn
+        self.n_accounts = n_accounts
+        self.init_amount = init_amount
+
+    async def initdb(self):
+        async with self.conn.cursor() as cur:
+            await cur.execute('DROP TABLE IF EXISTS bank_accs')
+            await cur.execute('CREATE TABLE bank_accs(uid int primary key, amount int)')
+            await cur.execute(
+                '''
```

Comment:
nit: the formatting is a bit odd here.

Reply:
I executed `yapf -r -i` from readme, and apparently it moved `'''` to the new line. I'll try to make it look better, but maybe code style check for python tests should be added to CI?

Reply:
>  but maybe code style check for python tests should be added to CI?

Yeah, that would've been so much better. Unfortunately, formatting/linting checks in CI will first require code cleanups.

Reply:
If someone decides to go for that, I'd vote for the `black` formatter.

Reply:
> If someone decides to go for that, I'd vote for the black formatter.

Sure, why not. I recently tried `poetry` and liked it, so I guess we could also replace pipenv.